src/components/PopupDelete.js

```diff
@@ -0,0 +1,29 @@
+import Popup from "./Popup.js";
+
+export default class PopupDelete extends Popup {
+  constructor(popupSelector, handleFormSubmit) {
+    super({ popupSelector });
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._submitButton = this._popupElement.querySelector(".modal__button");
+  }
+
+  setEventListeners() {
+    this._popupForm.addEventListener("submit", (e) => {
+      e.preventDefault();
+      this._submitCallback();
+    });
+    super.setEventListeners();
+  }
+
+  renderLoading(loading) {
+    if (loading) {
+      this._submitButton.textContent = "Deleting...";
+    } else {
+      this._submitButton.textContent = "delete";
```

Comment:
`"delete"`

The initial button text in `PopupDelete` is `Yes`

Reply:
* You haven't corrected my remark https://github.com/SalpStick/se_project_aroundtheus/pull/5#discussion_r1581305952